### PR TITLE
adb-logcat: add page

### PR DESCRIPTION
**Pull request by HUGHNew**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/adb-logcat.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display lines that match a regular expression:
+
+`adb logcat -e {{regular_expression}}`
+
+- Filter the log with filterspecs(`{{complete_tag:[V | D | I | W | E | F | S]}}`), `*` for all but not a wildcard(`*:S` suppresses others)
+
+`adb logcat {{tag0:V}} {{tag1:D}} *:S`
+
+- Color the log(usually use with filters)
```
Comment (kbdharun) [inline pages/common/adb-logcat.md:18]:
```suggestion
- Color the log(usually use with filters):
```

File: pages/common/adb-logcat.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# logcat
```
Comment (kbdharun) [inline pages/common/adb-logcat.md:1]:
```suggestion
# adb-logcat
```
Even though it is an extension of `logcat` , I think mentioning `adb-logcat` would be better

**Review by kbdharun — CHANGES_REQUESTED** [review]
Welcome to tldr @HUGHNew , I have requested some changes that need to be addressed.

**Comment by HUGHNew** [comment]
> Welcome to tldr @HUGHNew , I have requested some changes that need to be addressed.

Thanks for your advice.

**Review by kbdharun — APPROVED** [review]

File: pages/common/adb-logcat.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display system logs:
+
+`adb logcat`
+
+- Display lines that match a regular expression:
+
+`adb logcat -e {{regular_expression}}`
+
+- Filter the log with filterspecs(`{{complete_tag:[V | D | I | W | E | F | S]}}`), `*` for all but not a wildcard(`*:S` suppresses others)
```
[the remark this answers is not in the document]
Comment (CleanMachine1) [reply pages/common/adb-logcat.md]: I think we should change this as well. Its quite confusing.

File: pages/common/adb-logcat.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`adb logcat`
+
+- Display lines that match a regular expression:
+
+`adb logcat -e {{regular_expression}}`
+
+- Filter the log with filterspecs(`{{complete_tag:[V | D | I | W | E | F | S]}}`), `*` for all but not a wildcard(`*:S` suppresses others):
+
+`adb logcat {{tag0:V}} {{tag1:D}} *:S`
```
Comment (navarroaxel) [inline pages/common/adb-logcat.md:16]: I think we can split this into 3 examples to be easier to understand
```suggestion
- Display logs for a tag in a specific mode ([V]erbose, [D]ebug, [I]nfo, [W]arning, [E]rror, [F]atal, [S]ilent), filtering other tags:

`adb logcat {{tag}}:{{mode}} *:S`

- Display logs for React Native applications in [V]erbose mode [S]ilencing other tags:

`adb logcat ReactNative:V ReactNativeJS:V *:S`

- Display logs for all tags with priority level [W]arning and higher:

`adb logcat *:W`
```

File: pages/common/adb-logcat.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Display logs for React Native applications in [V]erbose mode [S]ilencing other tags:
+
+`adb logcat ReactNative:V ReactNativeJS:V *:S`
+
+- Display logs for all tags with priority level [W]arning and higher:
+
+`adb logcat *:W`
+
+- Color the log(usually use with filters):
```
Comment (CleanMachine1) [inline pages/common/adb-logcat.md:26]:
```suggestion
- Color the log (usually use with filters):
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/adb-logcat.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/adb-logcat.md:26]: I'll apply this tiny fix and merge it.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**